src/borg/remote.py

```diff
@@ -136,19 +205,30 @@ def serve(self):
                             tb = '%s\n%s' % (traceback.format_exc(), sysinfo())
                             logging.error(msg)
                             logging.log(tb_log_level, tb)
-                        exc = "Remote Exception (see remote log for the traceback)"
-                        os.write(stdout_fd, msgpack.packb((1, msgid, e.__class__.__name__, exc)))
+                        exc = 'Remote Exception (see remote log for the traceback)'
+                        if dictFormat:
+                            os.write(stdout_fd, msgpack.packb({b'i': msgid, b'exception_class': e.__class__.__name__, b'exception_args': e.args}))
+                        else:
+                            os.write(stdout_fd, msgpack.packb((1, msgid, e.__class__.__name__, exc)))
```

Comment:
excellent, this then also fixes the rpc-exception-args-transporting issue that I dropped from #1685